docker-compose.yml

```diff
@@ -1,73 +1,73 @@
 version: '3'
 services:
-  app:
```

Comment:
why you need to change the yaml ?

Reply:
It looks, like a sudden manually change, because YAML is ignored in [.prettierignore](https://github.com/open-condo-software/condo/pull/332/files#diff-b640b344ee7f3f03d2a443795a5d0708ef50e2e6e34214109ab2aad13ad6ba98)

Reply:
Force-pushed Prettier pass at `645776ca`.